### PR TITLE
Docs

### LINKED ISSUE #1548: Create simple streaming example
The example chart referenced in #1531 would make an ideal candidate for porting to d3fc as an example.

https://github.com/finallyayo/realtime-d3-graph/blob/master/app.js

### LINKED ISSUE #1553: Make old documentation available?
Hi! Love this module, it's proven invaluable in some of the work am picking up again after a hiatus. I came back and found that none of the old API documentation links work, and the lack of walk-through style guides that were available make it difficult to get to know the pieces of this API.

I was able to get the examples working locally so I could see the source in a reasonable way, but I had to pull down the repo, install everything, and then bundle the entire repository. This seems like a steep barrier to entry for anyone coming across d3fc for the first time who isn't already invested in using it.

Please consider making the old documentation available; I thought it was really user-friendly and showed off the power and flexibility of df3c, and the composability philosophy outlined in its documentation was one of the first things that drew me to it.

### PR DESCRIPTION
**Pull request by chrisprice**
Fixes #1553.
Fixes #1548.

### CODE REVIEW

**Comment by chrisprice** [comment]
I'd recommend reviewing it commit by commit, the last commit removes d3fc-site which makes things a bit noisy.

**Comment by ColinEberhardt** [comment]
- [x] 7338859
- [x] a35ea3a
- [x] e670e07
- [x] 0e0aae3
- [x] 26d7429
- [x] 52ce6c0
- [x] 0673b6c

**Review by ColinEberhardt — APPROVED** [review]
A few very minor comments, but otherwise looks really good 👍

File: README.md [hunk 1]
```diff
@@ -20,37 +20,65 @@ npm install d3fc
...
+* [d3fc-financial-feed](https://github.com/d3fc/d3fc/tree/master/packages/d3fc-financial-feed/README.md)
+* [d3fc-group](https://github.com/d3fc/d3fc/tree/master/packages/d3fc-group/README.md)
+* [d3fc-label-layout](https://github.com/d3fc/d3fc/tree/master/packages/d3fc-label-layout/README.md)
+* [d3fc-pointer](https://github.com/d3fc/d3fc/tree/master/packages/d3fc-pointer/README.md)
+* [d3fc-random-data](https://github.com/d3fc/d3fc/tree/master/packages/d3fc-random-data/README.md)
+* [d3fc-rebind](https://github.com/d3fc/d3fc/tree/master/packages/d3fc-rebind/README.md)
+* [d3fc-sample](https://github.com/d3fc/d3fc/tree/master/packages/d3fc-sample/README.md)
+* [d3fc-series](https://github.com/d3fc/d3fc/tree/master/packages/d3fc-series/README.md)
+* [d3fc-shape](https://github.com/d3fc/d3fc/tree/master/packages/d3fc-shape/README.md)
+* [d3fc-site](https://github.com/d3fc/d3fc/tree/master/packages/d3fc-site/README.md)
```
Comment (j-n-white) [inline README.md:52]: This links to the now deleted site package.

Comment (chrisprice) [reply README.md:52]: Good spot! Thanks, I'll get that fixed.

**Pull request closed without merging**